static/app/types/event.tsx

```diff
@@ -395,6 +395,7 @@ type EventBase = {
     version: string;
   } | null;
   sdkUpdates?: Array<SDKUpdatesSuggestion>;
+  startTimestamp?: number;
```

Comment:
I don't think we should mess with the base type. Can we use `EventTransaction` in our code instead? that's got startTimestamp baked in already.